### PR TITLE
Mc faq

### PR DESCRIPTION
**Pull request by joelmayemba**

### CODE REVIEW

**Review by chouaibj — APPROVED** [review]

File: src/Controller/FaqController.php [hunk 1]
```diff
@@ -13,6 +13,9 @@ class FaqController extends AbstractController
     public function index(FaqRepository $faqRepository): Response
     {
         $faq = $faqRepository->findAll();
-        return $this->render('faq/index.html.twig', ['pageName' => "FAQ", 'faq' => $faq,]);
+        return $this->render('faq/index.html.twig', ['pageName' => "Foire Aux Questions",
+        'pageDescription' => "Nos Mocktails",
```
Comment (Benja-san) [inline src/Controller/FaqController.php:17]: is it the faq description ?

File: src/Controller/MemberController.php [hunk 2]
```diff
@@ -17,6 +17,8 @@ public function index(MemberRepository $memberRepository): Response
         $member = $memberRepository->findAll();
         return $this->render('member/index.html.twig', [
             'pageName' => "Notre équipe",
+            'pageDescription' => "Nos Mocktails",
```
Comment (Benja-san) [inline src/Controller/MemberController.php:20]: Is it a placeholder ?

File: templates/faq/index.html.twig
Comment (Benja-san) [inline templates/faq/index.html.twig:1]: wrap your faq loop into an ul an wrap all faq inside an li

File: templates/member/index.html.twig [hunk 3]
```diff
@@ -17,23 +17,23 @@
 		<div class="container overflow-hidden">
 			<div class="row ">
```
Comment (Benja-san) [inline templates/member/index.html.twig:18]: wrap you members loop into an ul and put an li to start each member

File: templates/partials/_navbar.html.twig [hunk 4]
```diff
@@ -1,75 +1,37 @@
...
-                    </li>
-                </ul>
-            </div>
-        </div>
-    </div>
-</nav>
+<div class="navbar navbar-light bg-light navbar-expand-md px-3">
+	<a href="#" class="navbar-brand">
+		MOCKTAIL CELESTE
+		{# <img src="/static_files/images/logos/logo_1.png" alt="" width="50px" height="50px"/> #}
```
Comment (Benja-san) [inline templates/partials/_navbar.html.twig:4]: remove dead code

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Pull request merged**